test/sample_local_test.js

```diff
@@ -17,9 +17,9 @@ describe('BStack Local Testing', async function() {
   });
 
   it('check tunnel is working', async function () {
-    await driver.get('http://bs-local.com:45691/check');
-    let source = await driver.getPageSource();
-    assert(source.match(/Up and running/i) != null);
+    await driver.get('http://bs-local.com:45454/');
+    let title = await driver.getTitle();
+    assert(title.match(/BrowserStack Local/i) != null);
```

Comment:
This is fine but to keep this in sync with other repos let's match presence of string in the title rather than exact match..